### DISCUSSION TITLE
Is there a way to control the spacing between the bottom of the screen and the bottom of the brand image?

### DISCUSSION THREAD

**Discussion by dannyinmattr** [zoontek/react-native-bootsplash#625] [Q&A]
when running the CLI, it just defaults to a certain spacing between the brand image and the bottom of the screen, but I'd like to make this smaller

**Comment by zoontek** [comment]
No, it uses [`windowSplashScreenBrandingImage`](https://developer.android.com/develop/ui/views/launch/splash-screen#set-theme) under the hood on Android, and it cannot be moved.
